bff/fancy.py

```diff
@@ -162,6 +162,39 @@ def get_peaks(s: pd.Series, distance_scale: float = 0.04):
     return peaks_dates.values, peaks[1]['peak_heights']
 
 
+def idict(d: Dict[Any, Hashable]) -> Dict[Hashable, Any]:
+    """
+    Invert a dictionary meaning that keys will be become values and values will become keys.
+
+    Parameters
+    ----------
+    d : dict of any to hashable
+        Dictionary to invert
+
+    Returns
+    -------
+    dict of hashable to any
+        Inverted dictionary
+
+    Raises
+    ------
+    TypeError
+        If original values are not Hashable
+
+    Examples
+    -------
+    >>> idict({1: 4, 2: 5})
+    {4: 1, 5: 2}
+    """
+
+    try:
+        _ = set(d.values())
+    except TypeError:
+        raise TypeError(f'TypeError: values of dict {d} are not hashable.')
```

Comment:
A warning about lost values could be useful in case of duplicates values that will be lost.